content/en/logs/archives/reloading/s3.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Log Archive Reloading
+kind: documentation
+private: true
+beta: true
+aliases:
+  - /guides/archive_reloading
+further_reading:
+- link: "logs/explorer"
+  tag: "Documentation"
+  text: "See how to explore your logs"
+- link: "logs/archives/s3"
+  tag: "Documentation"
+  text: "Learn how to create an S3 log archive"
+---
+
+<div class="alert alert-warning">
+This feature is in private beta. Request early access by emailing support@datadoghq.com. Reloading is currently only supported for AWS S3 Archives, and for Datadog users of the Datadog US site.
+</div>
+
+Once you are [archiving the logs you manage in Datadog to your own S3 bucket][1], you can reload logs from your archives back into your Datadog account when you need them. This is especially useful if you wish to query or analize log data that is either (A) very old, or (B) had been excluded from indexing when it was originally sent to your Datadog account. Reloaded logs become avialable in your Log Explorer, and all its features--including search, patterns, and analytics--are available on reloaded log events just as if they were fresh, new log events. 
```

Comment:
Let's assume that someone might fall on this documentation page without even knowing anything about archives.

So let's have a brief explanation about what archives are and why they are important (log have a short retention, ..., maybe a link to Logging Without limits article and the Archive documentation).

And then talk about why anyone would need archives and then mention that you can reload logs from archives into Datadog.

Let's make sure it's not a sales pitch but a clear explanation of why you need archives and why reloading them can be useful.